addons/crm/views/crm_lead_views.xml

```diff
@@ -809,6 +870,44 @@ if record:
             <field name="action" ref="crm.crm_lead_all_leads"/>
         </record>
 
+        <!-- My Activities Menu -->
+        <record id="action_my_activities" model="ir.actions.act_window">
```

Comment:
Quick guidelining: if you read guidelines, this should have an id like crm_lead_action_my_activities . See guidelines for more details :) normally there are examples in it.